shared_model/backend/protobuf/common_objects/proto_common_objects_factory.hpp

```diff
@@ -0,0 +1,259 @@
+/**
```

Comment:
The whole file is implantation inside an .hpp, won't is cause troubles with recompiling etc? Isn't it better to split it?

Reply:
It will only be used in setup of storage, so its fine.